frontend/controller/actions/group.js

```diff
@@ -373,6 +373,33 @@ export default (sbp('sbp/selectors/register', {
       throw new GIErrorUIRuntimeError(L('Failed to rename chat channel.'))
     }
   },
+  'gi.actions/group/changeChatRoomDescription': async function (params: GIActionParams) {
+    try {
+      await sbp('gi.actions/chatroom/changeDescription', {
+        ...omit(params, ['options', 'contractID', 'data', 'hooks']),
+        contractID: params.data.chatRoomID,
+        data: {
+          description: params.data.description
+        },
+        hooks: {
+          prepublish: params.hooks?.prepublish,
+          postpublish: null
+        }
+      })
+
+      return await sbp('chelonia/out/actionEncrypted', {
+        ...omit(params, ['options', 'action', 'hooks']),
+        action: 'gi.contracts/group/changeChatRoomDescription',
+        hooks: {
+          prepublish: null,
+          postpublish: params.hooks?.postpublish
+        }
+      })
```

Comment:
Why are there two messages being sent?

Can you refactor this so that only `'gi.actions/chatroom/changeDescription` is called?

Reply:
The reason why I created this `gi.actions/group/changeChatRoomDescription` function is to call `gi.contracts/group/changeChatRoomDescription` function so that the group could save the attributes of its chatrooms.
The reason why the group should save the attributes of its chatrooms is that the group members should see the group chatrooms and their details even though they are not part of. It's not good to sync those chatrooms every time they are trying to see the details.

Reply:
OK, makes sense.

Reply:
OK, makes sense. Could you add a comment here explaining that then?

Reply:
Okay. Sure.